### DISCUSSION TITLE
csi_recv_router 这个代码是从路由获取csi数据，但连接路由是怎么连接的

### DISCUSSION THREAD

**Discussion by Lijianping77** [espressif/esp-csi#140] [General]

**Comment by MacChu0315-Espressif** [comment]
Are you experiencing issues with code usage or do you want to discuss the process of establishing a connection between devices and routers? 

If it's the former, you can configure the SSID and password of the router you want to connect to in the menuconfig.

If it's the latter, you can refer to this documentation: [ESP32-C3 Wi-Fi Station General Scenario](https://docs.espressif.com/projects/esp-idf/en/release-v5.1/esp32c3/api-guides/wifi.html#esp32-c3-wi-fi-station-general-scenario).
Please feel free to reach us for further assistance~
